wurst/systems/core/init/Units.wurst

```diff
@@ -0,0 +1,33 @@
+package Units
+import ClosureForGroups
+
+public destructable MAMMOTH_GATE
```

Comment:
If you rename this variable (which I agree is a good idea) make sure that you update the references to it in Respawn.wurst and Game.wurst.

Reply:
Or you can just remove it from this file and move the "finding" of this unit to Respawn.wurst (in that case I'd probably also add the initialization from Game.wurst for the gate to Respawn.wurst as well)